tests/services/test_entities_injection_service.py

```diff
@@ -83,8 +83,8 @@ def test_inject_entities_payload_has_correct_format(mqtt_mock, artist_data):
     artist_entity_name = "snips/artist"
 
     inject_entities = EntitiesInjectionService(hermes_host)
-    actual_json = inject_entities.build_payload(artist_entity_name, artist_data)
-
+    inject_entities.build_entities_payload(artist_entity_name, artist_data)
```

Comment:
You have a TODO at `line 62` where you can use a global variable.

Reply:
Yes and TODO is waiting for a task where all this data is going to be centralized in a file and imported when convenient. I think it is not this PR responsibility to deal with this.

Reply:
👍